WookieeMatt/Essence20
# Added more weapons

##### In this PR
- Added Automatic, Standard, Limited and Restricted Weapons
- Introduced fitting weapon icons
- Build new folder structure to organize weapon effects
- Added corresponding weapon effects 

##### Testing
- Verify folder structure
- Check new Icons
- Test new weapons
